pkg/hintrunner/zero/zerohint_others_test.go

```diff
@@ -33,15 +33,16 @@ func TestZeroHintMemcpy(t *testing.T) {
 					{Name: "continue_copying", Kind: uninitialized},
 				},
 				ctxInit: func(ctx *hinter.HintRunnerContext) {
-					ctx.ScopeManager.EnterScope(map[string]any{
-						"n": feltString("5"),
-					})
+					err := ctx.ScopeManager.AssignVariable("n", *feltString("5"))
```

Comment:
i guess you can use `feltInt64` or `feltUint64` instead of `feltString` here. Generally we use feltString for big numbers that dont feat in a uint64
it doesnt really matter but its better style and its more consistent with the rest of our code

Reply:
Right, Made the changes.